server/package.json

```diff
@@ -6,7 +6,7 @@
     "node": "*"
   },
   "dependencies": {
-    "@funboxteam/crafter": "4.1.0",
+    "@funboxteam/crafter": "file:./funboxteam-crafter-4.2.0.tgz",
```

Comment:
At now, I use local Crafter package with updates from https://github.com/funbox/crafter/pull/10

Thus, this PR cannot be merged right now. But it still can be reviewed.

Reply:
Published @funboxteam/crafter@4.1.1.

Reply:
I've replaced crafter with public version. It seems, bundled extension (locally packed) works as expected